phpcs.xml.dist

```diff
@@ -60,11 +60,13 @@
 		<exclude-pattern>*/src/IterableCodeExtractor\.php$</exclude-pattern>
 	</rule>
 
-	<!-- Whitelist variable names for classes that extend a PSR-2 package. -->
+	<!-- Whitelist property name for a class that extends a PSR-2 package. -->
```

Comment:
This documentation may need some tuning as well.

AFAICS the property comes from an object which is `json_decode()`d and not necessarily related to the parent class this class extends.